### PR TITLE
Dismemberment Rework

### PR DESCRIPTION
**Pull request by yackemflam**
:cl: Yackemflam
tweak: Dismemberment chance is now a var seperated from the old equation
Have fun with high dismemberment low force weapons
/:cl:

### CODE REVIEW

**Comment by Bawhoppen** [comment]
did you not learn when we said to use the actual git program

**Comment by yackemflam** [comment]
I didn't this is based off of an old pr

File: code/game/objects/items/weapons/melee/misc.dm [hunk 1]
```diff
@@ -23,11 +23,12 @@
 
 /obj/item/weapon/melee/sabre
```
Comment (GunHog) [inline code/game/objects/items/weapons/melee/misc.dm]: I believe this is an attempt to nerf this weapon again after being denied twice.

**Comment by Niknakflak** [comment]
After reading through, this is an awful awful PR that changes a shit ton of thing's chances of dismemberment based on your own personal view.

**Comment by yackemflam** [comment]
There, everything is now back to their original dismemberment chance.

**Comment by KorPhaeron** [comment]
You're never gonna fix conflicts in the web editor, come get help setting up git in #coderbus

**Pull request closed without merging**